cmd/termui/view/termuic/termuiRenders/widgetsRender.go

```diff
@@ -237,8 +238,10 @@ func (wr *WidgetsRender) prepareChainInfo(numMillisecondsRefreshTime int) {
 	numConnectedPeers := wr.presenter.GetNumConnectedPeers()
 	numLiveValidators := wr.presenter.GetLiveValidatorNodes()
 	numConnectedNodes := wr.presenter.GetConnectedNodes()
-	rows[8] = []string{fmt.Sprintf("Peers / Validators / Nodes: %d / %d / %d",
-		numConnectedPeers, numLiveValidators, numConnectedNodes)}
+	numIntraShardValidators := wr.presenter.GetIntraShardValidators()
```

Comment:
shouldn't we update `MetricNumIntraShardValidatorNodes` metric on heartbeat v1 as well? it will be 0 before hb v1 deactivation

Reply:
yup, that is true. I do not think it worth adding the code in the hb v1 just for this new metric. Since we are planning removing the hb v1 subsystem, anyway